### ISSUE TITLE
Unable to Open Pages Within a Channel with a Middle Click

### ISSUE THREAD

**Issue by moriel5** [lassekongo83/yt-classic-polymer#13]
The old, pre-polymer website supported opening pages within a channel, such as Videos, Playlists, About, etc, with a middle click on the respective tab, something that was taken away with the polymer theme.

I see that this functionality has not been replicated yet by the YouTube Classic Polymer Theme extension.

**Comment by lassekongo83** [comment]
I don't think this is possible. The tabs don't have any href attribute and are handled by YouTube's page manager.

**Comment by moriel5** [comment]
Yeah, I took a look, and you are right.
Hopefully someone will find a way in the future, I'll try asking the guys behind NewPipe-Extractor.
